src/ActionButtons/index.js

```diff
@@ -9,37 +9,49 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import React from 'react'
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react'
 import PropTypes from 'prop-types'
 import { ActionButton, Text } from '@adobe/react-spectrum'
 
 import Bug from '@spectrum-icons/workflow/Bug'
 import Edit from '@spectrum-icons/workflow/Edit'
 
-const ActionButtons = ({ gitUrl, filePath, branch, issues, ...props }) => (
+const ActionLink = ({ href, children }) => (
+  <a
+    href={href}
+    css={css`
+      text-decoration: none;
+    `}
+    target='_blank'
+    rel='noopener noreferrer nofollow'
+  >
+    {children}
+  </a>
+)
+
+const ActionButtons = ({
+  gitUrl = '',
+  filePath = '',
+  branch = '',
+  issues,
+  ...props
+}) => (
   <div {...props}>
-    <ActionButton
-      isQuiet
-      onPress={() => {
-        document.location.href = `${gitUrl}/edit/${branch}/${filePath}`
-      }}
-      aria-label='Edit page'
-    >
-      <Edit size='S' />
-      <Text>Edit this page</Text>
-    </ActionButton>
-    <ActionButton
-      isQuiet
-      onPress={() => {
-        issues
-          ? (document.location.href = issues)
-          : (document.location.href = `${gitUrl}/issues/new?body=Issue%20in%20${filePath}`)
-      }}
-      aria-label='Log issue'
+    <ActionLink href={`${gitUrl}/edit/${branch}/${filePath}`}>
+      <ActionButton isQuiet aria-label='Edit page' excludeFromTabOrder>
+        <Edit size='S' />
+        <Text>Edit this page</Text>
+      </ActionButton>
```

Comment:
Should we not expect a change in any snapshot tests?

Reply:
We need more and better tests. Right now everything passes. Someday, I'd love to go through this repo and do nothing but add tests and docs.